tls/s2n_client_extensions.c

```diff
@@ -429,3 +445,23 @@ static int s2n_recv_client_sct_list(struct s2n_connection *conn, struct s2n_stuf
     /* Skip reading the extension, per RFC6962 (3.1.1) it SHOULD be empty anyway  */
     return 0;
 }
+
+
+static int s2n_recv_client_max_frag_len(struct s2n_connection *conn, struct s2n_stuffer *extension)
+{
+    if (!conn->config->enable_server_mfl) {
+        fprintf(stderr, "warning: Maximum Fragmentation Length not enabled in S2N server, continuing with default max\n");
```

Comment:
So we don't want to let users know about invalid mfl input or mfl not enabled through fprints?
Are we notifying them through other channel?

'fprintf(stderr, "warning: Maximum Fragmentation Length not enabled in S2N server, continuing with default max\n");
 `
`fprintf(stderr, "warning: Invalid Maximum Fragmentation Length requested, continuing TLS handshake with default length\n");`